content/docs/rendering-elements.md

```diff
@@ -34,7 +34,7 @@ We call this a "root" DOM node because everything inside it will be managed by R
 
 Applications built with just React usually have a single root DOM node. If you are integrating React into an existing app, you may have as many isolated root DOM nodes as you like.
 
-To render a React element into a root DOM node, pass both to [`ReactDOM.render()`](/docs/react-dom.html#render):
+To render a React element, first create a React root with [`ReactDOM.createRoot()`](/docs/react-dom-client.html#createroot) and pass the element to `root.render()`:
```

Comment:
pass the *React* element.

The DOM element to createRoot, the React element to render.